.cirrus.yml

```diff
@@ -619,19 +619,12 @@ compose_test_task:
     gce_instance: *standardvm
     matrix:
       - env:
-            TEST_FLAVOR: compose
```

Comment:
Minor/Non-blocking: Should the now dead-code be stripped out of `contrib/cirrus/setup_environment.sh` and `contrib/cirrus/runner.sh`?

Reply:
nevermind, I should have scrolled down :relaxed:

Reply:
What dead-code is left?

Reply:
None, I was narrowly focused, didn't see it was removed.  We're all good here, nothing to see, move along, move along :rofl: